### PR TITLE
adding id params for specific routes so we can update through swagger

### PR DESCRIPTION
**Pull request by bmgdev**

### CODE REVIEW

File: server/api/accounts.js [hunk 1]
```diff
@@ -73,6 +73,13 @@ const register = function (server, serverOptions) {
             tags: ['api','accounts'],
             description: 'Get a customer account by ID. [Admin Scope]',
             notes: 'Get a customer account by ID.',
+            validate: {
+                params: {
+                    id : Joi.string()
+                            .required()
+                            .description('the id to get the account'),
```
Comment (jedireza) [inline server/api/accounts.js]: Can we put all these on a single line?

Comment (bmgdev) [reply server/api/accounts.js]: will make the change

**Comment by bmgdev** [comment]
updated onto one line, fixed comma dangles for linter

**Comment by jedireza** [comment]
Thank you!

**Pull request merged**